composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1",
-        "pimple/pimple": "^1.0",
```

Comment:
removing dependency is not minor

Reply:
In this case, it's a replacement and will still have Pimple available